source/read/retrieve.txt

```diff
@@ -0,0 +1,227 @@
+.. _pymongo-retrieve:
+
+=============
+Retrieve Data
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: code examples, read, search, cursor
+
+Overview
+--------
+
+In this guide, you can learn how to use {+driver-long+} to retrieve
+data from a MongoDB collection by using read operations. You can call the
+``find()`` method to retrieve documents that match a set of criteria.
```

Comment:
Since this is an overview for all of retrieve, I think we should mention both find methods that get talked about througout
```suggestion
``find()`` or the ``find_one()`` methods to retrieve documents that match a set of criteria.
```

Reply:
yep. tried to do that throughout but overlooked this one